src/custom/pages/Faq/Faq.md

```diff
@@ -0,0 +1,117 @@
+
+
+From only this January 2021 to now, the total amount of value extracted by miners (etc.) on Ethereum transactions has reached [$ 382.4 Million, including successful and failed transactions.](https://explore.flashbots.net/)
+
+### To what does the term Coincidence of Wants (CoWs) refer?
+
+Coincidence of Wants (CoWs) can be [explained](https://en.wikipedia.org/wiki/Coincidence_of_wants) as “an economic phenomenon where two parties each hold an item the other wants, so they exchange these items directly.” CowSwap facilitates CoWs among traders and their orders through using batch auctions as a core mechanism. 
+
+
+This means, on CowSwap, when two traders each hold an asset the other wants, a trade can be settled directly between them without an external market maker or liquidity provider. This leads to better prices for the individual traders (because traditionally market makers add a fee—referred to as spread—for their surface). 
+
+
+CowSwap allows for coincidence of wants (CoWs) orders to be traded directly against one another. Only the excess order amount that cannot be settled directly with other CowSwap traders are sent to the underlying AMMs (automated market makers). 
+
+### How am I protected from MEV (Arbitrage, Front running, Sandwiching) with CowSwap?
+CowSwap leverages batch auctions with uniform clearing prices for all trades in the same batch. Because of the uniform clearing price , there is no need for ordering the transactions within a single batch. Because everyone receives the same price across assets it’s not possible for _any_ value to be extracted by placing transactions in a certain order. This prevents the primary strategy used in MEV.
+
+
+Batches are decentrally settled by an external, independent party (called “solvers”) on-chain. Solvers are a person or entity who submits order settlement solutions that maximize trade surplus for a given batch. Solvers are incentivized to implement professional transaction management techniques that allow them to set very tight slippage bounds on any interactions with external liquidity sources (e.g. trading CoW excess on Uniswap). This dramatically reduces the manipulation surface of miners and front-runners.
+
+
+Additionally, depending on the composition of the orders submitted and valid for a given batch, the existence of CoWs may significantly reduce the amount that has to be exchanged via external MEV-prone protocols, such as Uniswap.
+
+### How does CowSwap determine prices?
+CowSwap settles batch auctions in discrete time intervals. In the absence of other traders, CowSwap matches traders against the best available Uniswap liquidity (note: other base-liquidity sources such as Balancer are being added soon).
```

Comment:
Either `are being added` or `will be added soon`

Reply:
@annamsgeorge correcting this small detail. It's reviewed text by Kia, I hope is fine.

Reply:
thank you for noticing and correcting!